.gitignore

```diff
@@ -1,2 +1,5 @@
 cache
 img-LinuxFr.org
+tests/
+go.mod
+go.sum
```

Comment:
If I understand correctly, these two files define how to build the project with exact version for dependencies.

They seems like `Gemfile` and `Gemfile.lock` in the Ruby world.

I would commit them because they are useful to have a more reproducible build.

Furthermore it would remove the need to give instruction in the REAMDE about how to build them.

Reply:
done